src/Model/Sluggable/SluggableMethods.php

```diff
@@ -42,6 +42,16 @@ private function getRegenerateSlugOnUpdate()
     }
 
     /**
+     * Returns whether or not the unique slug gets regenerated.
+     *
+     * @return bool
+     */
+    public function getRegenerateUniqueSlug()
```

Comment:
Yes, you're right and thanks for your suggestion!
I want to be consistent with `getRegenerateSlugOnUpdate` only.
Anyway, the `shouldGenerateUniqueSlugs` name will be better.